vcsim/main.go

```diff
@@ -42,6 +42,12 @@ import (
 	"github.com/vmware/govmomi/vim25/types"
 )
 
+// vcsim does not track versions like govc does, it only shows if a binary was
+// built during a tag event or from a git checkout.
+const Version = "dev"
```

Comment:
We could also move `govc/flags.Version` somewhere common and use it for both vcsim and govc.

Reply:
good point, maybe it's time to move version to a top-level package

Reply:
made the move, now it's shared amongst all the packages (including toolbox)